service-front/app/config/routes.php

```diff
@@ -95,13 +96,25 @@
     $app->get('/activate-account/{token}', Actor\Handler\ActivateAccountHandler::class, 'activate-account');
 
     // User auth
-    $app->route('/login', Actor\Handler\LoginPageHandler::class, ['GET', 'POST'], 'login');
+    $app->route('/login', [
+        new ConditionalRoutingMiddleware(
+            $container,
+            $factory,
+            $ALLOW_GOV_ONE_LOGIN,
+            function ($request, $handler) use ($container) {
+                return new \Laminas\Diactoros\Response\RedirectResponse('/home');
+            },
```

Comment:
```suggestion
            function ($request, $handler) {
                return new \Laminas\Diactoros\Response\RedirectResponse('/home');
            },
```

Forgot to deal with this yesterday. We no longer need the container in our anonymous function so this can be removed.

Reply:
We don't technically use the request or handler either so maybe try to remove those too and see what happens?

In something like Go this would break as the function then would not match the expected interface but I think PHP will just roll with it.